.yamllint

```diff
@@ -18,6 +39,24 @@ rules:
     # Allows a 10% overage from the default limit of 80
     max: 88
 
-  # yamllint doesn't like when we use yes and no for true and false,
-  # but that's pretty standard in Ansible.
-  truthy: disable
+  # Using anything other than strings to express octal values can lead to unexpected
+  # and potentially unsafe behavior. Ansible strongly recommends against such practices
+  # and these rules are needed for Ansible compatibility. Please see the following for
+  # more information:
+  # https://ansible.readthedocs.io/projects/lint/rules/risky-octal/
+  octal-values:
+    # Do not allow explicit octal values (those beginning with a leading 0o).
+    forbid-explicit-octal: true
+    # Do not allow implicit octal values (those beginning with a leading 0).
+    forbid-implicit-octal: true
```

Comment:
I'm fine with this as it is.  It's more explicit and clearer to the uninitiated reader to use `u=rw,g=r,o=r`, e.g., rather than `0644` or `0o644`.

Reply:
You can still use octal representations (and for Ansible modules we may need to), but they must be stored as string values instead of a direct octal representation. So for example it would be `"0644"` or `"0o644"`.